backend/de.metas.business.rest-api-impl/src/main/java/de/metas/rest_api/v2/pricing/PricesRestController.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * #%L
+ * de.metas.business.rest-api-impl
+ * %%
+ * Copyright (C) 2021 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.rest_api.v2.pricing;
+
+import de.metas.Profiles;
+import de.metas.common.pricing.v2.pricelist.request.JsonRequestPriceListVersionUpsert;
+import de.metas.common.pricing.v2.productprice.JsonRequestProductPriceUpsert;
+import de.metas.common.rest_api.v2.JsonResponseUpsert;
+import de.metas.util.web.MetasfreshRestAPIConstants;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import lombok.NonNull;
+import org.springframework.context.annotation.Profile;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequestMapping(value = {
+		MetasfreshRestAPIConstants.ENDPOINT_API_DEPRECATED + "/prices",
+		MetasfreshRestAPIConstants.ENDPOINT_API_V1 + "/prices",
+		MetasfreshRestAPIConstants.ENDPOINT_API_V2 + "/prices" })
```

Comment:
```suggestion
@RequestMapping(value = {
		MetasfreshRestAPIConstants.ENDPOINT_API_V2 + "/prices" })
```

even if technically we could have it in v1, i would like pleople to start directly with v2 when they start using new stuff